src/dragonball/src/dbs_virtio_devices/Cargo.toml

```diff
@@ -32,6 +32,7 @@ serde = "1.0.27"
 serde_json = "1.0.9"
 thiserror = "1"
 threadpool = "1"
+vm-device = "0.1.0"
```

Comment:
why using `vm-device` instead of `dbs-device`?

Reply:
Sorry, this was a misuse, I fixed it.